src/core/utils/EventManager.ts

```diff
@@ -98,7 +129,19 @@ export class EventManager {
                 // TODO: keydown should be matched with existing shortcuts. If
                 // it matches an intent shortcut, trigger the corresponding
                 // intent, otherwise do not trigger a 'keydown' intent.
-                return;
+                // CTRL+ALT
+                if (payload.ctrlKey && payload.altKey && !payload.shiftKey && !payload.metaKey) {
+                    // TODO: use payload.code (Digit[0-6]) instead of
+                    // payload.key
+                    if (Object.keys(ctrlAltShortcuts).includes(payload.key)) {
+                        name = ctrlAltShortcuts[payload.key].name;
+                        payload.value = ctrlAltShortcuts[payload.key].value;
+                    } else {
+                        return;
+                    }
+                } else {
+                    return;
+                }
```

Comment:
I think this is not rebased on master, as this part is missing the applyFormat part added by nby. Both this and the applyFormat shortcuts will need to be converted to use the keymap.

Reply:
Indeed.